src/emmy/ratio.cljc

```diff
@@ -12,7 +12,7 @@
   (:refer-clojure :exclude [ratio? numerator denominator rationalize])
   (:require #?(:clj [clojure.core :as core])
             #?(:clj [clojure.edn] :cljs [cljs.reader])
-            #?(:cljs ["fraction.js/bigfraction.js$default" :as Fraction])
+            #?(:cljs ["fraction.js/bigfraction.js" :as Fraction])
```

Comment:
interesting, why do JS people keep messing with their builds...

Reply:
I did notice that (1) in the current version, there's an exports block in fraction.js's package.json, (2) bigfraction.js  is not listed there, and (3) I still don't know why this was ever a problem on GitHub, where the older version had no exports clause and I guess was relying on a permissive interpretation.

I wanted to get "everyone onto the same page" and so experimented with adding and removing `$default`. I don't even know what it's really for, I was just trying to shake things into place. 

I'm quite open to the idea of just stashing bigfraction.js somewhere in the tree and forgetting about it